src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/cjsm/CjsmResource.kt

```diff
@@ -20,11 +20,11 @@ import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.ErrorRespon
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.config.ResourceNotFoundException
 import org.springframework.security.core.userdetails.UserDetails as PrincipalUserDetails
 
-@Hidden
 @RestController
 @RequestMapping(produces = [MediaType.APPLICATION_JSON_VALUE])
 class CjsmResource(private val cjsmService: CjsmService) {
 
+  @Hidden
```

Comment:
Just hide the refresh endpoint as it's not available outside the namespace anyway - but don't want to hide the new endpoint below